.github/ISSUE_TEMPLATE/06-other.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "\U00002753 Other"
+about: File an issue that doesn't fit into the other categories
+title: ""
+---
```

Comment:
should we add a needs-triage label?

Reply:
i was thinking it should go the other way and we assign a `triaged` label once issues have been sent to the relevant people, that way nothing is missed

Reply:
i support that!